src/System.Management.Automation/engine/MergedCommandParameterMetadata.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
@bergmeister, your last commit had 1380 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

NullCoalesceOperations.Null conditional assignment operator ??=.Variable doesnot exist
```powershell
Expected $null, because 1, but got 1.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1: line 23
23:             $variableDoesNotExist | Should -Be 1
```
Replace Operator.Replace operator.Replace operator can be case-insensitive and case-sensitive
```powershell
Expected exactly $null, because Cook, but got 'Cook'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/ReplaceOperator.Tests.ps1: line 16
16:             $res | Should -BeExactly "Cook"
```
Replace Operator.Replace operator.Replace operator can take 2 arguments, a mandatory pattern, and an optional substitution
```powershell
Expected exactly $null, because PowerShell, but got 'PowerShell'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/ReplaceOperator.Tests.ps1: line 27
27:             $res | Should -BeExactly "PowerShell"
```
Replace Operator.Replace operator substitutions.Replace operator supports numbered substitution groups using `$n
```powershell
Expected exactly $null, because Tld of 'domain.example' is - 'example', but got 'Tld of 'domain.example' is - 'example''.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/ReplaceOperator.Tests.ps1: line 37
37:             $res | Should -BeExactly "Tld of 'domain.example' is - 'example'"
```
NullCoalesceOperations.Null conditional assignment operator ??=.Rhs is a cmdlet
```powershell
Expected exactly $null, because Where-Object, but got 'Where-Object'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1: line 59
59:             $x.Definition | Should -BeExactly 'Where-Object'
```